File: Python/2808.py
x, y = input().split()

col_x, line_x = x
col_y, line_y = y

col_diff = abs(ord(col_x) - ord(col_y))
line_diff = abs(int(line_x) - int(line_y))

if (col_diff == 1 and line_diff == 2) or (col_diff == 2 and line_diff == 1):
    print('VALIDO')
else:
    print('INVALIDO')

File: JavaScript/2808.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const [x, y] = lines.shift().split(' ');

const [col_x, line_x] = x;
const [col_y, line_y] = y;

const col_diff = Math.abs(col_x.charCodeAt(0) - col_y.charCodeAt(0));
const line_diff = Math.abs(parseInt(line_x) - parseInt(line_y));

if ((col_diff === 1 && line_diff === 2) || (col_diff === 2 && line_diff === 1)) {
    console.log('VALIDO');
} else {
    console.log('INVALIDO');
}